imports/api/genes/hectar/addHectar.js

```diff
@@ -57,10 +57,10 @@ class HectarProcessor {
       // Filters undefined data (with a dash) and splits into an array for
       // comma-separated data.
       for (const [key, value] of Object.entries(annotations)) {
-        if (value[0] === '-') {
+        if (value === undefined || value[0] === '-') {
```

Comment:
Any possibility that a value might be defined, but as an empty string ("")?

Reply:
should not

Reply:
Alright, merging